frontends/election-manager/src/components/printable_area.tsx

```diff
@@ -3,17 +3,21 @@ import ReactDom from 'react-dom';
 
 interface Props {
   children?: React.ReactNode;
+  dataTestId?: string;
```

Comment:
You can allow the typical style for this by defining the prop like so:

```suggestion
  'data-testid'?: string;
```

Then you can do this:

```ts
export function PrintableArea({ 'data-testid': dataTestId, children }: Props): JSX.Element {
```